### PR TITLE
add Traditional Chinese

### PR DESCRIPTION
**Pull request by breakwa11**

### CODE REVIEW

File: shadowsocks-csharp/shadowsocks-csharp.csproj [hunk 1]
```diff
@@ -271,6 +271,7 @@
     <Content Include="Data\cn.txt" />
     <Content Include="Data\privoxy_conf.txt" />
     <Content Include="Data\user-rule.txt" />
+    <None Include="Data\zh_tw.txt" />
```
Comment (wongsyrone) [inline shadowsocks-csharp/shadowsocks-csharp.csproj]: Shouldn't this be "Content"?

**Comment by breakwa11** [comment]
fixed

File: shadowsocks-csharp/Data/zh_tw.txt [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+Load Balance=負載均衡
+High Availability=高可用
+Choose by statistics=根據統計
+
+# Config Form
+
+&Add=添加(&A)
+&Delete=刪除(&D)
+Server=伺服器
+Server Addr=伺服器地址
```
Comment (wongsyrone) [inline shadowsocks-csharp/Data/zh_tw.txt]: It seems they prefer "伺服器位址". What's your opinion?

File: shadowsocks-csharp/Data/zh_tw.txt [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+Server Port=伺服器端口
```
Comment (wongsyrone) [inline shadowsocks-csharp/Data/zh_tw.txt]: "伺服器連接埠" seems more native.

File: shadowsocks-csharp/Data/zh_tw.txt [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+Password=密碼
+Encryption=加密
+Proxy Port=代理端口
```
Comment (wongsyrone) [inline shadowsocks-csharp/Data/zh_tw.txt]: Ditto.

File: shadowsocks-csharp/Data/zh_tw.txt [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+New server=未配置的伺服器
+Move &Up=上移(&U)
+Move D&own=下移(&O)
+
+# Proxy Form
+
+Edit Proxy=代理設置
+Use Proxy=使用代理
+Proxy Addr=代理地址
+Proxy Port=代理端口
```
Comment (wongsyrone) [inline shadowsocks-csharp/Data/zh_tw.txt]: Ditto.

File: shadowsocks-csharp/Data/zh_tw.txt [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+Start on Boot=開機啟動
+SOCKS5 Proxy...=SOCKS5代理設置...
+Allow Clients from LAN=允許來自局域網的連接
+Local PAC=使用本地 PAC
+Online PAC=使用在線 PAC
+Edit Local PAC File...=編輯本地 PAC 文件...
+Update Local PAC from GFWList=從 GFWList 更新本地 PAC
+Edit User Rule for GFWList...=編輯 GFWList 的用戶規則...
+Show QRCode...=顯示二維碼...
+Scan QRCode from Screen...=掃描屏幕上的二維碼...
```
Comment (wongsyrone) [inline shadowsocks-csharp/Data/zh_tw.txt]: "螢幕"

File: shadowsocks-csharp/Data/zh_tw.txt [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+Edit Proxy=代理設置
+Use Proxy=使用代理
+Proxy Addr=代理地址
+Proxy Port=代理端口
+
+# Log Form
+
+&File=文件(&F)
+&Open Location=在資源管理器中打開(&O)
```
Comment (wongsyrone) [inline shadowsocks-csharp/Data/zh_tw.txt]: “資料總管”

**Comment by wongsyrone** [comment]
Please check comments above.

**Comment by breakwa11** [comment]
U r right, fixed again

**Comment by wongsyrone** [comment]
Thanks, merged.

**Pull request merged**